dali/c_api/c_api.cc

```diff
@@ -12,19 +12,77 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include "dali/c_api.h"  // NOLINT [build/include]
```

Comment:
Why moving to the top?

Reply:
I thought it's our way, to include the header file corresponding to the cc file on top of the include list?